src/widgets/range-input/range-input.js

```diff
@@ -41,8 +41,8 @@ const renderer = ({
   const shouldAutoHideContainer = autoHideContainer && rangeMin === rangeMax;
 
   const values = {
-    min: minValue !== -Infinity && minValue !== rangeMin ? minValue : null,
-    max: maxValue !== Infinity && maxValue !== rangeMax ? maxValue : null,
+    min: minValue !== -Infinity && minValue !== rangeMin ? minValue : undefined,
+    max: maxValue !== Infinity && maxValue !== rangeMax ? maxValue : undefined,
```

Comment:
Why `undefined` over `null`?

Reply:
`undefined` over `null` in order to avoid to pass `null` to the `refine` function of the `connector`. `null` will be treat as an invalid value instead of a reset action.

Reply:
Good for me.